rosdep/python.yaml

```diff
@@ -6670,7 +6670,7 @@ python3-pycryptodome:
   debian: [python3-pycryptodome]
   fedora: [python3-pycryptodomex]
   gentoo: [dev-python/pycryptodome]
-  openembedded: [python3-pycryptodome@meta-python]
+  openembedded: [python3-pycryptodomex@meta-python]
```

Comment:
I can't find this package in meta-python: https://layers.openembedded.org/layerindex/branch/master/layer/meta-python

It seems to only be in meta-python2: https://layers.openembedded.org/layerindex/recipe/114445/

Reply:
That's because in gatesgarth release it was moved from meta-python to oe-core, see:

https://git.openembedded.org/meta-openembedded/commit/?id=a8f3c00d8d113b46a49584682e10435157d516ca
https://git.openembedded.org/openembedded-core/commit/?id=a96f815c53364b119b5743b8b7100eb5588d5cf5

So @meta-python is correct for LTS release dunfell and for newer releases @openembedded-core would be more correct, but luckily these @ suffixes aren't used for anything (dropped by superflore), so it doesn't matter.